horqrux/primitives/parametric.py

```diff
@@ -7,18 +7,18 @@
 from jax import Array
 from jax.tree_util import register_pytree_node_class
 
-from ._misc import default_complex_dtype
-from .matrices import OPERATIONS_DICT
-from .noise import NoiseProtocol
-from .primitive import Primitive
-from .utils import (
+from .._misc import default_complex_dtype
```

Comment:
I tend to prefer absolute paths when the module is outside the current folder but that might be a bit stylistic.